src/top/data/objectron_detection.py

```diff
@@ -50,6 +50,7 @@ def decode(example, feature_names: List[str] = []):
     translation = example['object/translation'].reshape(num_instances, 3)
     orientation = example['object/orientation'].reshape(num_instances, 9)
     scale = example['object/scale'].reshape(num_instances, 3)
+    visibility = example['object/visibility'].reshape(num_instances, 1)
```

Comment:
Please rebase from `origin/main` since I don't want to review the same changes twice. Currently the common ancestor is `b756a1393eaf021eefeb2e5feeed4f77ad1fcdb4` which is 2 commits older.